client/src/css/App.css

```diff
@@ -150,4 +150,4 @@
 	100%{
 		-moz-transform:rotate(360deg)
 	}
-}
\ No newline at end of file
+}
```

Comment:
it looks great. Is this by accident?

Reply:
Awesome! And yes yes. This is where I was testing out the CSS.